tests/integration/unit_test/test_unit_test_dotnet8.py

```diff
@@ -0,0 +1,50 @@
+# dotnet unit tests are executed from test directories, so code_directories point to test directories
+from tests.integration.unit_test.unit_test_base import UnitTestBase
+
+
+class UnitTest_dotnet8_cookiecutter_aws_sam_hello_dotnet(UnitTestBase.DotNetCoreUnitTestBase):
+    directory = "dotnet8/hello"
+    code_directories = ["test/HelloWorld.Test"]
```

Comment:
Same as above, we can disable linting for now

Reply:
same response as above